src/index.js

```diff
@@ -1,14 +1,19 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./App";
+import App from "./App.js";
 import "./index.css";
 import state from "./state";
 
+
+var currentCategory = [];
+function changeCategory(category) {
+  return currentCategory.push(category);
```

Comment:
When you click the category links in the header it is not changing what products are shown in the web page.
you are missing one small thing from the changeCategory function. double check the specs for that
* Create a function in index.js called changeCategory thats takes a parameter category. It will assign category to currentCategory and call render()